masterlist.yaml

```
@@ -21503,6 +21503,24 @@ plugins:
         itm: 30
         udr: 14
 
+  - name: 'WSL_Redone.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/39832/'
+        name: 'Knights of the White Stallion Redone'
+      - link: 'https://www.nexusmods.com/oblivion/mods/51426/'
+        name: 'Knights of the White Stallion Redone FIX'
+    inc:
+      - 'Neo-White Stallion Lodge Mod.esp'
+      - '300_White Stallion 4.esp'
```

These just need to be sorted:
```suggestion
      - '300_White Stallion 4.esp'
      - 'Neo-White Stallion Lodge Mod.esp'
```